server/bastionlab_polars/src/lib.rs

```diff
@@ -86,6 +90,53 @@ impl BastionLabPolars {
         }
     }
 
+    fn update_log(
+        self: &BastionLabPolars,
+        new_row: DataFrame,
+        ) -> Result<(), Status> {
+        let mut dfs = self.dataframes.write().unwrap(); 
+        let artifact = dfs.get_mut(LOG_DF);
+        match artifact{
+            Some(artifact) => { //log dataframe exists, append new_row and save
+                artifact.dataframe = artifact.dataframe.vstack(&new_row).unwrap_or(artifact.dataframe.clone());
+                drop(dfs);
+                return self.persist_df(LOG_DF);
+            }
+            None => {
+                let pol = Policy{
+                safe_zone: True,
+                unsafe_handling: Log,
+                };
+                let df = DataFrameArtifact::new(new_row, pol, Vec::new());
+                dfs.insert(LOG_DF.to_string(), df);
+                drop(dfs);
+                return self.persist_df(LOG_DF);
+            }
+        }    
+    }
+
+    fn update_log_value(
```

Comment:
What is this method doing?
Ideally, you just need to append new lines to the log
I can't see a situation that requires to update a past log, it seems to go against the very point of having logs